### PR TITLE
Add fuzzing tests for EventScanner

### PR DESCRIPTION
**Pull request by evgeny-stakewise**

### CODE REVIEW

File: sw_utils/event_scanner.py [hunk 1]
```diff
@@ -55,9 +55,6 @@ def __init__(
     ):
         self.processor = processor
         self.argument_filters = argument_filters
-        self._contract_call = lambda from_block, to_block: getattr(
-            processor.contract.events, processor.contract_event
-        ).get_logs(argument_filters=argument_filters, fromBlock=from_block, toBlock=to_block)
```
Comment (evgeny-stakewise) [inline sw_utils/event_scanner.py:60]: Replaced lambda function with regular method `_contract_call`

File: sw_utils/event_scanner.py [hunk 2]
```diff
@@ -69,7 +66,7 @@ async def process_new_events(self, to_block: BlockNumber) -> None:
         if current_from_block >= to_block:
             return
 
-        while current_from_block < to_block:
+        while current_from_block <= to_block:
```
Comment (evgeny-stakewise) [inline sw_utils/event_scanner.py:69]: Edge case. Tests catch this error when setting small chunk size.

File: sw_utils/tests/test_event_scanner.py [hunk 3]
```diff
@@ -47,3 +53,99 @@ async def test_basic(self):
...
+
+class SimpleEventProcessor(EventProcessor):
+    contract_event = 'event'
+
+    @staticmethod
+    async def get_from_block() -> BlockNumber:
+        return 0
+
+    @staticmethod
+    async def process_events(events: list[EventData], to_block: BlockNumber) -> None:
```
Comment (evgeny-stakewise) [inline sw_utils/tests/test_event_scanner.py:79]: Problem: method is static. I can't save results to instance attributes. I have to save somewhere else.

**Review by cyc60 — APPROVED** [review]
